app/controllers/v0/preferences_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module V0
+  class PreferencesController < ApplicationController
```

Comment:
Thinking out loud, since we are returning a `Preference` and its associated `PreferenceChoice`'s from this endpoint, I wonder if we should re-think the name of this controller.  Because we are not returning just a `Preference`, as the Rails & RESTful convention typically would for a `preference#show` action?  🤔

What do you think?

Reply:
I don't think we should rename the controller - it's in line with the model and actions within as we are now returning an index of `Preference` objects - wdyt?

Reply:
Based on all of the new contract work we've done since then, I'm good leaving as is 👍